### ISSUE TITLE
Push devices registrations are not clear with the "close other active session" button on the profile page.

### ISSUE THREAD

**Issue by nixxou** [RocketChat/Rocket.Chat#18541]
### Description:

Push devices registrations are not clear with the "close other active session" button on the profile page.
### Steps to reproduce:

1. Use the Rocket chat app on multiple android device with your Admin account
2. Login on your desktop, go to your profile, click on the "close other active session" button
3. Go to the Push notification admin page, click on the Test push button. You will see a green box on the top right that says it was send to multiples devices

### Expected behavior:

Rocket chat should not be able to send push if you force disconect all your mobile devices

### Actual behavior:

![image](https://user-images.githubusercontent.com/45721836/90067691-c1816a80-dcef-11ea-891a-5fa7053fe6c0.png)

### Server Setup Information:

- Version of Rocket.Chat Server: 3.4.2
- Operating System: Ubuntu 18 LTS
- Deployment Method:  snap
- Number of Running Instances: 1

### Client Setup Information

- Desktop App or Browser Version: Browser
- Operating System: Windows

**Comment by nixxou** [comment]
Ok, so i kinda solved the issue but it's defitivly something to fix. A simple user will not have the knowledge or the tools to do the same.

The problem i was facing is related to this issue :  https://github.com/RocketChat/Rocket.Chat/pull/18442

So i had to  : 
1- Increase the log level and monitor log to get the list of the push tokens of the registered devices.
2- login using the rest api, and remove manually each push token registered with a command like that
curl -H "X-Auth-Token: MYAUTHTOKEN"  -H "X-User-Id: MYUSERID"   -H "Content-type:application/json"  -X "DELETE"   http://localhost:3000/api/v1/push.token -d '{"token": "MYPUSHTOKEN"}'

And ... that didn't work and i was still having this kind of errors in my logs after reconnecting only one mobile device : 
server.js:204 Push ➔ error Error sending push to gateway (3 try) -> Error: failed [500] {"code":111,"error":"internal error sending notification to gcm"

 So after hitting my head on a wall for a moment, i just thought about rebooting the server and then it was ok.

Bottom line, if you click on "Loggout from other Logged in location", it should remove token of registered devices too.
